resolwe/flow/managers/listener/metrics.py

```diff
@@ -0,0 +1,179 @@
+"""Report the listener metrics to the monitoring system."""
+
+import logging
+import os
+from time import time
+from typing import Any
+
+from opentelemetry import metrics
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+from opentelemetry.sdk.resources import Resource
+
+from resolwe.flow.executors.socket_utils import (
+    Message,
+    MessageProcessingCallback,
+    MessageProcessingEventType,
+)
+
+logger = logging.getLogger(__name__)
+
+# How often to export metrics (in milliseconds).
+METRICS_EXPORT_INTERVAL = 60000
```

Comment:
We typically use granularity of 30s for other metrics.

Reply:
Changed to 30 seconds.